python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -935,25 +935,6 @@ def spawn_child(self, name, runtime_env):
         ray.shutdown()
 
 
-@pytest.mark.skipif(
-    os.environ.get("CI") and sys.platform != "linux",
-    reason="This test is only run on linux CI machines.",
-)
-def test_runtime_env_logging_to_driver(ray_start_regular_shared, log_pubsub):
```

Comment:
Duplicated to https://github.com/ray-project/ray/blob/ee1711fe418e44818ff791b266cd9dea42a65957/python/ray/tests/test_runtime_env.py#L418